unix/man-section-7.md

```diff
@@ -0,0 +1,61 @@
+# Man Section 7
+
+For most of us, `man` pages are primarily the source of documentation for various system-level tools, be it `curl`, `ssh`, `gcc` or multitude of others.
+
+But that's not all that there is. TIL about `man` section 7 and some of its pages that you can't visit by mere accident.
+
+As `man 7 intro` puts it:
+
+```
+intro - Introduction to overview, conventions, and miscellany section
+
+Section  7  of  the  manual  provides  overviews  on  various  topics, and
+describes conventions and protocols, character set standards, the standard
+file system layout, and miscellaneous other things.
+```
+
+You can see the entire list for yourself with the following command:
+
+```
+apropos '.*' | grep '(7.* - ' | sort | less
+```
+
+Here is the overview of some pages that stood out to me. To visit a page, type `man <name>` or `man 7 <name>` to avoid collisions with pages in other sections.
+
+| page            | description
+| :-------------: | :----------
+| ascii           | ASCII table right in your `man` pager. With no extra tools. In 3 different layouts!
+| boot            | Fantastic and rather condensed explanation of Linux boot sequence.
```

Comment:
I suggest we just make it clear up front that this TIL applies to Linux distro X. I'm not sure TIL warrants any cross-platform effort. In fact, it may detract from the actual point: "`man 7` is cool."